freeipa/src/main/java/com/sequenceiq/freeipa/configuration/EventBusMetricConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.sequenceiq.freeipa.configuration;
+
+import java.util.concurrent.ExecutorService;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.sequenceiq.cloudbreak.logger.concurrent.MDCCleanerThreadPoolExecutor;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.jvm.ExecutorServiceMetrics;
+
+@Configuration
+public class EventBusMetricConfig {
+
+    @Inject
+    private MeterRegistry meterRegistry;
+
+    @Inject
+    @Named("eventBusThreadPoolExecutor")
+    private MDCCleanerThreadPoolExecutor executor;
+
+    @Bean
+    public ExecutorService eventBusThreadPoolExecutorService() {
+        return ExecutorServiceMetrics.monitor(meterRegistry, executor, "eventBusThreadPoolExecutor", "reactor");
```

Comment:
maybe can be easier to track if the name contains the service name instead of the same name (`eventBusThreadPoolExecutor`)? Or is it required?

Reply:
In the end it will appear in the metric as a tag, I think it is more useful to use the executor name.

Reply:
hm, what will the difference between the sdx, freeipa, environment, redbeams or core executor service in the metrics?